src/docs/guides/database-migration-guide.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Migrating from Legacy Plugins to Database Services
+---
+
+We've introduced a more secure and flexible way to deploy databases on Railway.  The purpose of this guide is to provide an overview of why we are making these changes, what they mean for you, and how you can migrate.
+
+## What is Changing?
+### Legacy Plugins
+In the early stages of Railway, we introduced `plugins`, a foundational part of our offering. They were databases deployed from Docker images with a fixed version, no data persistence, and no way to customize the database to your needs.
+
+### Database Services
+Some of our newest features - Volumes, Docker Image Deployments, TCP Proxy - open the door for deploying databases as services, complete with data persistence, flexible configuration and management, and connectivity from the external network.
+
+## Why Migrate?
```

Comment:
Can maybe also include
- Updated versions of database images
- Ability to use the private network

Reply:
let me know what you think of the update @coffee-cup and whether that's what you had in mind